GoogleUtilities/Example/GoogleUtilities.xcodeproj/xcshareddata/xcschemes/Example_iOS.xcscheme

```diff
@@ -39,16 +39,6 @@
                ReferencedContainer = "container:GoogleUtilities.xcodeproj">
             </BuildableReference>
          </TestableReference>
```

Comment:
Why deleting IntegrationTests?

Reply:
There is no such a target in the project. It was marked as "missed"